edenai_apis/features/llm/llm_interface.py

```
@@ -72,3 +72,68 @@ def llm__chat(
             provider_params (dict, optional): Additional parameters specific to the provider
         """
         raise NotImplementedError
+
+    @abstractmethod
+    async def llm__achat(
+        self,
+        messages: List = [],
```

Ruff (0.12.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)